app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,36 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.lookup_for_authentication_with identifier: email
+
+    if user.authenticate password
+      session.update user_id: user.id
+      redirect_to root_url, notice: t("session.success", email: user.email)
+    else
+      flash.now[:error] = t("session.error")
+      render :new
+    end
+  end
+
+  def destroy
+    session.update user_id: nil
```

Comment:
I think `reset_session` should works too http://api.rubyonrails.org/classes/ActionController/Base.html#M000668